tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -1,266 +1,272 @@
-                  INVISIBLE                                                        -- invisible during typechecking
-                  Annotation
-
-  Variance      = STABLE                                                           -- invariant
-                | COVARIANT
-                | CONTRAVARIANT
-
-  Annotation    = ANNOTATION     Length tycon_Type fullAnnotation_Term             -- An annotation, given (class) type of constructor, and full application tree
-
-Note: The signature of a SELECTin or TERMREFin node is the signature of the selected symbol,
-      not the signature of the reference. The latter undergoes an asSeenFrom but the former
-      does not.
-
-Note: Tree tags are grouped into 5 categories that determine what follows, and thus allow to compute the size of the tagged tree in a generic way.
-
-  Category 1 (tags 1-59)   :  tag
-  Category 2 (tags 60-89)  :  tag Nat
-  Category 3 (tags 90-109) :  tag AST
-  Category 4 (tags 110-127):  tag Nat AST
-  Category 5 (tags 128-255):  tag Length <payload>
-
-
-Standard-Section: "Positions" LinesSizes Assoc*
-
-  LinesSizes    = Nat Nat*                 // Number of lines followed by the size of each line not counting the trailing `\n`
-
-  Assoc         = Header offset_Delta? offset_Delta? point_Delta?
-                | SOURCE nameref_Int
-  Header        = addr_Delta +              // in one Nat: difference of address to last recorded node << 3 +
-                  hasStartDiff +            // one bit indicating whether there follows a start address delta << 2
-                  hasEndDiff +              // one bit indicating whether there follows an end address delta << 1
-                  hasPoint                  // one bit indicating whether the new position has a point (i.e ^ position)
-                                            // Nodes which have the same positions as their parents are omitted.
-                                            // offset_Deltas give difference of start/end offset wrt to the
-                                            // same offset in the previously recorded node (or 0 for the first recorded node)
-  Delta         = Int                       // Difference between consecutive offsets,
-  SOURCE        = 4                         // Impossible as header, since addr_Delta = 0 implies that we refer to the
-                                            // same tree as the previous one, but then hasStartDiff = 1 implies that
-                                            // the tree's range starts later than the range of itself.
-
-All elements of a position section are serialized as Ints
-
-
-Standard Section: "Comments" Comment*
-
-  Comment       = Length Bytes LongInt      // Raw comment's bytes encoded as UTF-8, followed by the comment's coordinates.
-
-
-**************************************************************************************/
+/** Notation:
```

Comment:
I'm a bit relunctant to reformat the whole comment since it makes it harder to use git blame to determine when a particular tag was added (although there are ways around that: https://www.moxio.com/blog/43/ignoring-bulk-change-commits-with-git-blame), is the reformatting actually needed or could we just add the code block around the existing comment?

Reply:
wow, TIL `blame.ignoreRevsFile`. I need that on legacy Scala 2! BTW currently all my TILs are from the Scala ecosystem. I wonder if you can ignore revs while bisecting.

Reply:
Maybe leave the original comment text sans special comment with double star, then add a real doc comment. The pretty comment would have to be kept in sync with the ugly feature database comment.

Reply:
Alright I have a much smaller diff now, although I'm not a bit confused. So the missing part actually seemed to be `@syntax markdown` at the bottom, which I didn't realized before. Without it no matter what the code blocks looked all smushed and even seemed to just miss some of the blocks:

<img width="1153" alt="Screenshot 2021-12-31 at 10 26 43" src="https://user-images.githubusercontent.com/13974112/147816469-95db7450-8ee2-487e-bbc1-bcb3953a3128.png">

After adding the `@syntax markdown` at the bottom, everything worked.

<img width="1161" alt="Screenshot 2021-12-31 at 10 42 08" src="https://user-images.githubusercontent.com/13974112/147816499-ef950277-8c57-4361-90b6-3d476e8e3f6b.png">

However looking at the [docs](https://dotty.epfl.ch/docs/usage/scaladoc/docstrings.html) it says:

> @syntax <name> let you change the parser for docstring. The default syntax is markdown, however you can change it using this directive. Currently available syntaxes are markdown or wiki, e. g. @syntax wiki

So I don't get why markdown is needed here if that's the default?

Reply:
> So I don't get why markdown is needed here if that's the default?

No idea, perhaps the tasty project is compiled with syntax wiki by default for some reason? /cc @BarkingBad